src/components/WorkspaceAdd/WorkspaceAddModal.tsx

```diff
@@ -14,6 +14,7 @@ import {
 import { ExclamationCircleIcon } from '@patternfly/react-icons/dist/esm/icons/exclamation-circle-icon';
 import type { K8sModelCommon } from '@openshift/dynamic-plugin-sdk-utils';
 import { k8sCreateResource, K8sResourceCommon } from '@openshift/dynamic-plugin-sdk-utils';
+// import { useNavigate } from 'react-router-dom';
```

Comment:
We should probably just remove this logic as it will never be readded

Reply:
Same with the below commented logic

Reply:
If PR 25 is merged first, then I'll comment on this logic so the user is taken to the details page after adding a new workspace.

Reply:
Since we are merging #25 first, I removed the comments to enable the functionality of directing the user to the details page after creation.